fabric-common/types/index.d.ts

```diff
@@ -225,9 +224,14 @@ export class DiscoveryService extends ServiceAction {
 	public getDiscoveryResults(refresh?: boolean): Promise<any>;
 	public close(): void;
 }
+export interface RegistrationOpts {
+	startBlock?: number|string|Long;
+	endBlock?: number|string|Long;
+	unregister?: boolean;
+}
 
 export class EventListener {
-	constructor(listenerType: string, callback: any, options: any, event: string);
+	constructor(eventService: EventService, listenerType: string, callback: any, options?: RegistrationOpts, event?: RegExp|string, chaincodeId?: string);
```

Comment:
I don't think this should be included in the public API

Reply:
Or could we keep this while we do not to export this class to outside? like what pure js do.